timur/lib/client/jsx/components/timur_nav.jsx

```diff
@@ -39,6 +39,7 @@ const getTabs = () => ({
   map: Routes.map_path(CONFIG.project_name),
   manifests: Routes.manifests_path(CONFIG.project_name),
   plots: Routes.plots_path(CONFIG.project_name),
+  'view editor': Routes.views_path(CONFIG.project_name),
```

Comment:
I would favor calling the tab `views` for consistency. Also since only admins can see it can we somehow suppress the link for non-admins?